test/transforms/dec128.test.js

```diff
@@ -22,52 +24,101 @@ const doesNotChangeNumbers = {
     "(1 + (4 - 2)) * (3 + 4);",
 };
 
+const constructor = {
+  "works with plain number": {
+    code: "Decimal(1);",
+    output: `${libName}(1);`
+  },
+  "works with string": {
+    code: 'Decimal("1");',
+    output: `${libName}("1");`
+  },
+  "passes expression through": {
+    code: 'Decimal(1 + 1);',
+    output: `${libName}(1 + 1);`
+  },
+  "throws on null": {
+    code: "Decimal(null)",
+    error: "TypeError: Can't convert null or undefined to Decimal."
+  },
+  "throws on explicit undefined": {
+    code: "Decimal(undefined)",
+    error: "TypeError: Can't convert null or undefined to Decimal."
+  },
+  "throws on implicit undefined": {
+    code: "Decimal()",
+    error: "TypeError: Can't convert null or undefined to Decimal."
+  },
+  "coerces boolean true": {
+    code: "Decimal(true);",
+    output: `${libName}(1);`
+  },
+  "coerces boolean false": {
+    code: "Decimal(false);",
+    output: `${libName}(0);`
+  },
+  "coerces BigInt": {
+    code: "Decimal(1n);",
+    output: `${libName}("1");`
+  },
```

Comment:
I think I'd add `Decimal(1.5m)` and `Decimal(Decimal(1))` as test cases.

Reply:
Good suggestion!